avr8.rkt

```diff
@@ -181,14 +182,16 @@
    [`(ori ,(? avr-reg-hi? src) ,(? avr-imm-b? b))           (dw (+ #x6000 (avr-reg-hi-d src) (avr-imm-b b)))]
    [`(andi ,(? avr-reg-hi? src) ,(? avr-imm-b? b))          (dw (+ #x7000 (avr-reg-hi-d src) (avr-imm-b b)))]
    
+   [`(sbci ,(? avr-reg-hi? src) ,(? avr-imm-b? b))          (dw (+ #x4000 (avr-reg-hi-d src) (avr-imm-b b)))]
+   
    [`(swap ,(? avr-reg? reg)) (dw (+ #x9402 (avr-reg-d reg)))]
    [`(asr ,(? avr-reg? reg))  (dw (+ #x9405 (avr-reg-d reg)))]
    [`(lsr ,(? avr-reg? reg))  (dw (+ #x9406 (avr-reg-d reg)))]
    [`(ror ,(? avr-reg? reg))  (dw (+ #x9407 (avr-reg-d reg)))]
    
    [`(xch z ,(? avr-reg? reg)) (dw (+ #x9204 (avr-reg-d reg)))]
    [`(las z ,(? avr-reg? reg)) (dw (+ #x9205 (avr-reg-d reg)))]
-   [`(lac z ,(? avr-reg? reg)) (dw (+ #x9006 (avr-reg-d reg)))]
+   [`(lac z ,(? avr-reg? reg)) (dw (+ #x9206 (avr-reg-d reg)))]
```

Comment:
good call with this fix - I'm not sure what got me there, as I clearly saw this instruction in the grouping with other #x920x instructions but put #x900x and I have no idea why 😓